lms/templates/ccx/coach_dashboard.html

```diff
@@ -155,4 +170,24 @@
       $('#ccx_std_list_messages')[0].focus();
     }
   });
+  function validateForm(form) {
+    var newCCXName = $(form).find('#ccx_name').val();
+    var $errorMessage =  $('#ccx-create-message');
+    var hasCcxConnector = ${"true" if course and hasattr(course, 'ccx_connector') and course.ccx_connector else "false"};
+
+    if ((!newCCXName || newCCXName === "") && !hasCcxConnector) {
+        $errorMessage.text("${_('Please enter a valid CCX name.')}");
+        $errorMessage.show();
+        console.log("error");
```

Comment:
Let me ask this another way. Why are we logging to console? If this is meant to be a meaningful error log, it should say something more than just "error"